commons/addons/mongodb.md

```diff
@@ -21,6 +21,33 @@ The version currently installed by the add-on is :
 - on shared plans (Peanut) : MongoDB 3.2.9
 - on newly created dedicated databases (plans Hazelnut and above) : MongoDB 3.6.4
 
+## About Free Dababases
+
+Free plans are meant for tests and development usage only. Using these databases in production is not recomanded, because actual performances may vary depending on the global cluster's usage. Also, before switching to production, consider to upgrade to a dedicated database for better performances.
```

Comment:
- [ ] tests -> test.
- [ ] recomanded  -> recommended
- [ ] cluster's usage -> usage of the cluster
- [ ] consider to upgrade -> consider upgrading